Multisig/UI/Settings/OwnerKeyManagement/AddKeyAsNewOwner/AddOwnerController.swift

```diff
@@ -65,7 +65,7 @@ class AddOwnerController: UINavigationController, UIAdaptivePresentationControll
                                                                 owner: key,
                                                                 oldOwnersCount: safe.ownersInfo?.count ?? 0,
                                                                 oldThreshold: Int(safe.threshold ?? 0),
-                                                                newThreshold: 5)
+                                                                newThreshold: Int(safe.threshold ?? 0))
```

Comment:
how is the old and new threshold different? We should get the new threshold from the previous screen and use it here

Reply:
My bad, new it's solved